config-controller/api/v1alpha1/policy_types.go

```diff
@@ -18,11 +18,200 @@ package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/stackrox/rox/generated/storage"
 )
 
 // PolicySpec defines the desired state of Policy
 type PolicySpec struct {
-	Policy string `json:"policy,omitempty"`
+	Name        string   `json:"name,omitempty"`
+	Description string   `json:"description,omitempty"`
+	Rationale   string   `json:"rationale,omitempty"`
+	Remediation string   `json:"remediation,omitempty"`
+	Disabled    bool     `json:"disabled,omitempty"`
+	Categories  []string `json:"categories,omitempty"`
+	// +kubebuilder:validation:Enum=DEPLOY;BUILD;RUNTIME
+	LifecycleStages []string    `json:"lifecycleStages,omitempty"`
+	Exclusions      []Exclusion `json:"exclusions,omitempty"`
+	Scope           []Scope     `json:"scope,omitempty"`
+	// +kubebuilder:validation:Enum=UNSET_SEVERITY;LOW_SEVERITY;MEDIUM_SEVERITY;HIGH_SEVERITY;CRITICAL_SEVERITY
+	Severity string `json:"severity,omitempty"`
+	// +kubebuilder:validation:Enum=UNSET_ENFORCEMENT;SCALE_TO_ZERO_ENFORCEMENT;UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT;KILL_POD_ENFORCEMENT;FAIL_BUILD_ENFORCEMENT;FAIL_KUBE_REQUEST_ENFORCEMENT;FAIL_DEPLOYMENT_CREATE_ENFORCEMENT;FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT
+	EnforcementActions []string             `json:"enforcementActions,omitempty"`
+	Notifiers          []string             `json:"notifiers,omitempty"`
+	PolicyVersion      string               `json:"policyVersion,omitempty"`
+	PolicySections     []PolicySection      `json:"policySections,omitempty"`
+	MitreAttackVectors []MitreAttackVectors `json:"mitreAttackVectors,omitempty"`
+	CriteriaLocked     bool                 `json:"criteriaLocked,omitempty"`
+	MitreVectorsLocked bool                 `json:"mitreVectorsLocked,omitempty"`
+	IsDefault          bool                 `json:"isDefault,omitempty"`
+}
+
+func (p PolicySpec) ToProtobuf() *storage.Policy {
```

Comment:
Same comment here: Add the new field conversion to proto in this function.

Reply:
[a57f884](https://github.com/stackrox/stackrox/pull/12647/commits/a57f88401ac1563a5862cdc6cd37319cb3de609c)